app/controllers/spree/admin/resource_controller_decorator.rb

```diff
@@ -0,0 +1,22 @@
+Spree::Admin::ResourceController.class_eval do
```

Comment:
No, never override resource controller like this. This controller is used by all admin controllers.
let's discuss this once tomorrow